deploy/k8s/charts/trustification/values.yaml

```diff
@@ -212,4 +216,13 @@ modules:
     image: {}
     rust: {}
 
+  report:
```

Comment:
It looks like that's a duplicate. There seems to be another section before the modules section. I think there should only be one.

Reply:
That's the pattern used for other modules as well, e.g.

```
bombastic:
  bucket: bombastic
  topics:
    stored: sbom-stored
    failed: sbom-failed
    indexed: sbom-indexed


modules:

  bombasticApi:
    enabled: true
    ingress: {}
    image: {}
    infrastructure: {}
    tracing: {}
    metrics: {}
    rust: {}
```

Reply:
Maybe the first one can be renamed to `reportServer`?

Reply:
I see, yes that might make it more clear. Although I would assume that the "module" would be the `reportServer`.